app/assets/javascripts/discourse/app/lib/sidebar/topics-section/my-posts-section-link.js

```diff
@@ -0,0 +1,65 @@
+import I18n from "I18n";
+import { tracked } from "@glimmer/tracking";
+
+import BaseSectionLink from "discourse/lib/sidebar/topics-section/base-section-link";
+
+const USER_DRAFTS_CHANGED_EVENT = "user-drafts:changed";
+
+export default class MyPostsSectionLink extends BaseSectionLink {
+  @tracked draftCount = this.currentUser.draft_count;
+
+  constructor() {
+    super(...arguments);
+    this.appEvents.on(USER_DRAFTS_CHANGED_EVENT, this, this._updateDraftCount);
```

Comment:
Have to rely on this to update the drafts count now because we can't mark `currentUser.draft_count` as tracked until `model/user.js` is rewritten as a native class and glimmer componnet is made available in production.